sklearn/metrics/_plot/roc_curve.py

```diff
@@ -14,25 +20,37 @@ class RocCurveDisplay(_BinaryClassifierCurveDisplayMixin):
 
     It is recommend to use
     :func:`~sklearn.metrics.RocCurveDisplay.from_estimator` or
-    :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` to create
+    :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` or
+    :func:`~sklearn.metrics.RocCurveDisplay.from_cv_results` to create
     a :class:`~sklearn.metrics.RocCurveDisplay`. All parameters are
     stored as attributes.
 
-    Read more in the :ref:`User Guide <visualizations>`.
+    For more about the ROC metric, see :ref:`roc_metrics`.
+    For more about scikit-learn visualization classes, see :ref:`visualizations`.
```

Comment:
This adds reference the user guide section on roc metrics and visualization classes.

I initially wanted to follow the 'Read more in our user guide' line that we use everywhere else, but since we are reference 2 places in the user guide, i thought it would be okay to change for this case but I am not 100%. Happy to amend.